tests/test_forms.py

```diff
@@ -1,369 +0,0 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from __future__ import unicode_literals
-
-from rasa_core.events import (
-    ActionExecuted,
-    BotUttered,
-    SlotSet,
-    UserUttered
-)
-from rasa_core.actions.forms import (
-    BooleanFormField,
-    EntityFormField,
-    FormAction,
```

Comment:
FormActions are now part of the pySDK right, in which case we should remember to move these tests over there